Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/exchange/transform/uddi/UDDITransform.java

```diff
@@ -215,50 +215,50 @@ private static void buildTargetRegions(BusinessEntity entity, OrganizationType o
         }
     }
 
-    private static List<String> buildEmails(Contact contact) {
-        List<String> emails = null;
+    private static void buildEmails(Contact contact, ContactType cType) {
+        List<String> emails;
         if (contact != null && CollectionUtils.isNotEmpty(contact.getEmail())) {
             emails = new ArrayList<>();
             for (Email obj : contact.getEmail()) {
                 emails.add(obj.getValue());
             }
+            cType.getEmail().addAll(emails);
```

Comment:
Question: wonder what happen at runtime if the list is empty

Reply:
you will have an empty tag <email/>